forch/faucetizer.py

```diff
@@ -60,7 +45,9 @@ def process_device_placement(self, eth_src, placement, static=False):
                 if removed:
                     LOGGER.info('Removed %s device: %s', device_type, eth_src)
 
-    @_write_behavioral_config()
+            if not self._config.faucetize_interval_sec:
```

Comment:
if condition should be moved into a function rather than replicating 3x, and then rename function something like "_flush_behavorial_config()" -- since it's not strictly always writing it.  If you need both behaviors, then you could either 1) create another intermediate function, or 2) add an optional force=True argument for the cases when you always want to write it.

Reply:
Done